modules/tinymce/src/plugins/textpattern/test/ts/atomic/FindBlockPatternsTest.ts

```diff
@@ -3,7 +3,10 @@ import { getPatternSet } from 'tinymce/plugins/textpattern/api/Settings';
 import { findPattern } from 'tinymce/plugins/textpattern/core/BlockPattern';
 
 UnitTest.test('atomic.tinymce.plugins.textpattern.FindBlockPatternsTest', function () {
-  const patternSet = getPatternSet({});
+  const mockEditor = {
+    getParam: () => ({})
```

Comment:
@HAFRMO btw, the build looks to be failing still because this should be returning undefined or similar here to match the old behaviour. Here's the logs:
```
Run Tests / macFirefox / macos firefox / firefox-macos.atomic.tinymce.plugins.textpattern.FindBlockPatternsTest.modules/tinymce/src/plugins/textpattern/test/ts/atomic/FindBlockPatternsTest.ts (from firefox-macos)

Stacktrace
TypeError: actual is null

Logs:
```